components/dashboard/src/projects/NewProject.tsx

```diff
@@ -106,10 +106,24 @@ export default function NewProject() {
     }
 
     const reconfigure = () => {
+        //track project button clicked 'add_organisation'
+        trackProjectButton({
+            button: 'add_organisation',
+            button_context: undefined,
+            button_foreign_id: undefined
+        });
         openReconfigureWindow({
             account: selectedAccount,
             onSuccess: (p: { installationId: string, setupAction?: string }) => {
                 updateReposInAccounts(p.installationId);
+                //track 'organisation_authorised'
```

Comment:
heads up, that organization might be already authorized and we've just updated the selection of repos.

Reply:
thanks for the heads up, keeping it this way for now due to the lack of a better name.